CenterForOpenScience/cos.io
# [EOSF-822] Add block for TOP Committee members

## Ticket

https://openscience.atlassian.net/browse/EOSF-822

## Purpose

To display the TOP Committee members on the cos.io site (currently not displaying anything)

## Changes

- Add options in the People Block dropdown for `concise-top`, which will display the TOP-committee members
- Altered the model to fetch people with TOP-committee tags on render
- Added a conditional to the `people_block` template to render the TOP committee members
- Created new migrations for the altered model

The committee member blocks were formatted to look like the ambassadors.  Information displayed is the name and occupation.

## QA

The TOP Committee members will not show up on staging unless the `concise-top` selection is chosen on that page's block.  You'll need to have someone with access to staging admin go in and select it.
<img width="1247" alt="screen shot 2017-08-24 at 1 17 01 pm" src="https://user-images.githubusercontent.com/19379783/29679099-d6a68864-88ce-11e7-89d2-8b91ebace766.png">


After it's been changed in admin, the committee members should display.  You'll need to go to https://staging-cosio.herokuapp.com/our-services/top-guidelines/ and click on the last side-tab `Coordinating Committee` to see the changes.

## Screenshot
<img width="1354" alt="screen shot 2017-08-23 at 4 57 12 pm" src="https://user-images.githubusercontent.com/19379783/29638493-6b7891e4-8825-11e7-86a5-bab4e19e6e89.png">
